<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_save</class>
 <widget class="QDialog" name="Dialog_save">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="11" column="0">
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>Need Remind?</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Important?</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="3">
    <widget class="QLineEdit" name="lineEdit_4"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>From</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="2">
    <widget class="QTimeEdit" name="timeEdit_3"/>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QDateEdit" name="dateEdit"/>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>To</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Loop</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="2">
    <widget class="QPushButton" name="pushButton_ChooseLoop">
     <property name="text">
      <string>Choose</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Reminder time</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Character</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QDateEdit" name="dateEdit_2"/>
   </item>
   <item row="5" column="4">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>To</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QTimeEdit" name="timeEdit_2"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>P.S.</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="4">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>From</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QTimeEdit" name="timeEdit"/>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Location</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="3">
    <widget class="QLineEdit" name="lineEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QCheckBox" name="checkBox_3">
     <property name="text">
      <string>Done?</string>
     </property>
    </widget>
   </item>
   <item row="12" column="5">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
